gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -136,7 +136,6 @@ internal open class AnvilPlugin : KotlinCompilerPluginSupportPlugin {
     if (variant.variantFilter.syncGeneratedSources) {
       val isIdeSyncProvider = project.providers
         .systemProperty("idea.sync.active")
-        .forUseAtConfigurationTime()
```

Comment:
Does this not need to be replaced by a new api call?

Reply:
It was deprecated in Gradle 7.4 though, which is quite old now. Does that seem ok?

Reply:
If it's deprecated and there's not equivalent replacement or workaround from Gradle then removing it seems reasonable. But we should verify the new behavior and add a note in the changelog if that new restriction exists now.